packages/valaxy/node/cli/new.ts

```diff
@@ -0,0 +1,39 @@
+import type { Argv } from 'yargs'
+
+export const newPost = (cli: Argv<{}>) => {
+  cli.command(
+    'new <title>',
+    'Draft a new post',
+    (args) => {
+      args.usage('$0 <title> -p [path] -l [layout]')
+        .positional('title', {
+          describe: 'The title of the new post',
+          required: true,
```

Comment:
`t` alias for `title`?

Reply:
Since this is a positional parameter I don't see how alias is going to do with this. The usage is 

```shell
valaxy new 'post-title-blabla' ...other params.
```

Reply:
I got it.